src/helm.ts

```diff
@@ -1,6 +1,7 @@
 export const PREVIEW_SCHEME = 'helm-template-preview';
 export const PREVIEW_URI = PREVIEW_SCHEME + '://preview';
 export const INSPECT_SCHEME = 'helm-inspect-values';
+export const HELM_OUTPUT_PARSING_REGEX = /\t+/g;
```

Comment:
Do we need the separate regexes?  I would have thought that `\s+` would still work for tab-delimited output... or oh, can Helm output include spaces within a column?

Reply:
The `UPDATED` column in helm output contains a timestamp which contains spaces hence the regex with tabs is needed here as compared to `kubectl` output.